drivers/sensor/bm160/bm160.h

```diff
@@ -384,11 +483,10 @@ struct bmi160_range {
 };
 
 struct bmi160_device_config {
-#if defined(CONFIG_BMI160_TRIGGER)
 	const char *gpio_port;
 	gpio_pin_t int_pin;
 	gpio_dt_flags_t int_flags;
-#endif
+	u8_t chipid;
```

Comment:
Why is the trigger ifdef removed?

Reply:
Hard to define conditional compilation in macro expansion 

#define BMI160_INIT(idx)						\
									\
static const struct bmi160_device_config bmi160_config_##idx = {	\
	.chipid = BMI160_CHIP_ID,					\
	.gpio_port = DT_INST_GPIO_LABEL(idx, int_gpios),		\
	.int_pin = DT_INST_GPIO_PIN(idx, int_gpios),			\
	.int_flags = DT_INST_GPIO_FLAGS(idx, int_gpios),		\
};